adafruit_l3gd20.py

```diff
@@ -254,13 +265,20 @@ class L3GD20_SPI(L3GD20):
     :param baudrate: spi baud rate default is 100000
     """
 
-    def __init__(self, spi_busio, cs, rng=L3DS20_RANGE_250DPS, baudrate=100000):
+    def __init__(
+        self,
+        spi_busio,
+        cs,
+        rng=L3DS20_RANGE_250DPS,
+        rate=L3DS20_RATE_100HZ,
```

Comment:
Add rate after baudrate to prevent breaking code that relies on position of these args. It wouldn't matter if there was a ` *,` before the kwargs to make them kwarg-only.